### PR TITLE
Expand supported profiler docs with rendering of output files

### PR DESCRIPTION
**Pull request by jfy133**
In addition to the table, given some outputs don't actually have the headernames - therefore it might be useful for the user to have a visual example of what the output looks like

### CODE REVIEW

File: docs/contributing/supporting_new_profiler.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ modules need to be placed in a package of their own named for the profiler, for
 What do those three new modules need to do? Basically, taxpasta needs to be able
 to read a taxonomic profile, validate its correctness as much as possible, and
 finally transform it to what we call the
-[`StandardProfile`][taxpasta.domain.model.StandardProfile]. All
-further processing and logic is based on the [`StandardProfile`][taxpasta.domain.model.StandardProfile] so you don't need
+[`StandardProfile`][taxpasta.domain.model.standardprofile]. All
+further processing and logic is based on the [`StandardProfile`][taxpasta.domain.model.standardprofile] so you don't need
```
Comment (jfy133) [inline docs/contributing/supporting_new_profiler.md:22]:
```suggestion
[`StandardProfile`][taxpasta.domain.model.StandardProfile]. All
further processing and logic is based on the [`StandardProfile`][taxpasta.domain.model.Standard)rofile] so you don't need
```

File: docs/contributing/supporting_new_profiler.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ You could then continue either with the reading or transformation part of the sc
 We will begin with the reader so that you can get some real data into Python.
 
 For reading a taxonomic profile from a new tool, you need to inherit a new class
-from the abstract [`ProfileReader`][taxpasta.application.service.ProfileReader]. This
+from the abstract [`ProfileReader`][taxpasta.application.service.profilereader]. This
```
Comment (jfy133) [inline docs/contributing/supporting_new_profiler.md:111]:
```suggestion
from the abstract [`ProfileReader`][taxpasta.application.service.ProfileReader]. This
```

File: docs/contributing/supporting_new_profiler.md [hunk 3]
```diff
@@ -169,7 +169,7 @@ class Kraken2ProfileStandardisationService(ProfileStandardisationService):
 
 Finally, we need to transform the specific taxonomic profile into our standard
 profile. Similarly to the profile reader, there exists an abstract
-[`ProfileStandardisationService`][taxpasta.application.service.ProfileStandardisationService]
+[`ProfileStandardisationService`][taxpasta.application.service.profilestandardisationservice]
```
Comment (jfy133) [inline docs/contributing/supporting_new_profiler.md:172]:
```suggestion
[`ProfileStandardisationService`][taxpasta.application.service.ProfileStandardisationService]
```

File: docs/contributing/supporting_new_profiler.md [hunk 4]
```diff
@@ -197,7 +197,7 @@ The `pa.check_types` decorator validates the class method's input and output
 using the type annotations and the defined schema models.
 
 The `transform` class method itself needs to modify the given `pandas.DataFrame` such that
-the returned result looks like the [`StandardProfile`][taxpasta.domain.model.StandardProfile].
+the returned result looks like the [`StandardProfile`][taxpasta.domain.model.standardprofile].
```
Comment (jfy133) [inline docs/contributing/supporting_new_profiler.md:200]:
```suggestion
the returned result looks like the [`StandardProfile`][taxpasta.domain.model.StandardProfile].
```

File: docs/contributing/supporting_new_profiler.md [hunk 5]
```diff
@@ -256,7 +256,7 @@ from .kraken2 import (
 ### 5. Enable Support
 
 Finally, to make your profiler fully available to taxpasta, you have to add your profiler name
-to the [`SupportedProfiler`][taxpasta.infrastructure.application.SupportedProfiler] enumeration.
+to the [`SupportedProfiler`][taxpasta.infrastructure.application.supportedprofiler] enumeration.
```
Comment (jfy133) [inline docs/contributing/supporting_new_profiler.md:259]:
```suggestion
to the [`SupportedProfiler`][taxpasta.infrastructure.application.SupportedProfiler] enumeration.
```

File: docs/contributing/supporting_new_profiler.md [hunk 6]
```diff
@@ -271,7 +271,7 @@ class SupportedProfiler(str, Enum):
 ```
 
 Once that entry is made. Add your reader and standardisation service to the
-[`ApplicationServiceRegistry`][taxpasta.infrastructure.application.ApplicationServiceRegistry].
+[`ApplicationServiceRegistry`][taxpasta.infrastructure.application.applicationserviceregistry].
```
Comment (jfy133) [inline docs/contributing/supporting_new_profiler.md:274]:
```suggestion
[`ApplicationServiceRegistry`][taxpasta.infrastructure.application.ApplicationServiceRegistry].
```

**Review by Midnighter — APPROVED** [review]
Cheers!

**Pull request closed without merging**